src/CapabilityService.Tests/Builders/HttpClientBuilder.cs

```diff
@@ -39,9 +40,10 @@ public HttpClientBuilder WithService<TService>(TService serviceInstance)
         }
 
         private IWebHostBuilder CreateWebHostBuilder()
-        {
+        {
+			//TODO: Create test start that inherits from webapi startup

```

Comment:
Didnt you already do this by letting the StubStartUp inherit from the webapi startup class?